CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing to WAZP
+
+## Introduction
+
+**Contributors to WAZP are absolutely encouraged**, whether to fix a bug, develop a new feature, or improve the documentation.
+If you're unsure about any part of the contributing process, please get in touch. It's best to reach out in public, e.g. by opening an issue so that others can benefit from the discussion.
+
+## Contributing code
+
+### Creating a development environment
+
+It is recommended to use [conda](https://docs.conda.io/en/latest/) to install a development environment for
+WAZP. Once you have `conda` installed, the following commands
+will create and activate a `conda` environment with the requirements needed
+for a development environment:
+
+```sh
+conda create -n wazp-dev -c conda-forge python=3 pytables
+conda activate wazp-dev
+```
+
+This installs packages that often can't be installed via `pip`, including
+[hdf5](https://www.hdfgroup.org/solutions/hdf5/).
+
+To install WAZP for development, clone the GitHub repository, and then run from inside the repository:
+
+```sh
+pip install -e '.[dev]'
+```
+
+This will install the package, its dependencies,
+and its development dependencies.
+
+### Pull requests
+
+In all cases, please submit code to the main repository via a pull request. We recommend, and adhere, to the following conventions:
+
+- Please submit _draft_ pull requests as early as possible to allow for discussion.
+- One approval of a PR (by a repo owner) is enough for it to be merged.
+- Unless someone approves the PR with optional comments, the PR is immediately merged by the approving reviewer.
+- We typically merge via "Squash and Merge" on GitHub to maintain a clean commit history.
```

Comment:
```suggestion
```
To my taste we should just make this the rule.

I've just enforced this in settings (and therefore we don't need a sentence ☝️). In case you want to revert, it's the general settings tab at the bottom just before the danger zone under "Pull requests".

Reply:
![Screenshot 2023-03-06 at 10 40 10](https://user-images.githubusercontent.com/1836192/223087490-84f8a3b6-334a-4dd4-a4ab-532acaa1fe93.png)